content/docs/formal-verification/_index.md

```diff
@@ -0,0 +1,137 @@
+first game captures the analyzed protocol, and the final game is one where the required security property is obvious.
+The output is given as a bound on the probability of a successful attack as a function of the security parameters of the
+individual primitives used, and the number of protocol executions. A key benefit of the computational model is that this
+mimics how cryptographers usually prove security of cryptographic protocols. This makes it easier to carry over
+constructions from traditional pen-and-paper proofs to automated proofs.
+
+The computational model is clearly more expressive than the symbolic model. However, it is also presupposes a deep
+understanding of provable security and game-based proofs, which is not required when working in the symbolic model.
+
+Ultimately, which tool to use depends as much on your background as on the protocol you are trying to analyze. If you
+don't have a background in cryptography or formal verification we recommend that you start out with a simple symbolic
+verification tool like Verifpal. If you struggle to model your protocol in Verifpal, or need to express more complex
+security properties that are not supported by the tool, we suggest switching to a more expressive and mature symbolic
+prover like ProVerif or Tamarin. Finally, if you want to bound attack probabilities or translate a game-based proof from
+a paper, you need to work in the computational model with a tool like CryptoVerif.
+
+{{< hint info >}}
+
+### Where do I start?
+
+[Verifpal](https://verifpal.com) is a great starting point if you have no previous exposure to formal verification. It
+has a small set of built-in primitives that cover a range of use-cases, and the syntax mimics how developers typically
+think about cryptographic protocols. Verifpal's intuitive syntax also makes the resulting models easier to parse and
+maintain for less experienced users.
+
+This is an important point. If you develop a formal model of a protocol that is too complex for the developers of the
+protocol to maintain, then the proof becomes a snapshot in time with limited usefulness. If developers understand the
+model and can update and extend it as the protocol develops, it can be built on to provide assurance for future versions
+of the protocol as well.
+
+{{< /hint >}}
+
+{{< hint info >}}
+
+### How do I start?
+
+Since most cryptographic protocols have a large number of valid states, formal verification tools often struggle because
+of state-space explosion. For this reason, it is generally a good idea to try to divide your protocol into smaller
+components, or sub-protocols, like _registering a new device_ or _sending and receiving a single message_, and try to
+prove that these individual components provide the expected security guarantees.
+
+This approach clearly limits what the attacker can do since each sub-protocol is analyzed in a vacuum. However, it makes
```

Comment:
This could also be a great place to highlight the complementarity of tools. In the computation model we can find composability results allowing to easily break down protocols in subprotocol withouth worrying about composition. So if such a composition result exist it's nice then to focus symbolic effort on a subprotocol without worrying either.